### DISCUSSION TITLE
Issues with Docker container

### DISCUSSION THREAD

**Discussion by v-anne** [freelawproject/courtlistener#4459] [General]
I am attempting to run courtlistener locally to test a feature I'm working on.

Earlier in the week, I had no issue, but since yesterday I'm getting the following error: `psycopg.ProgrammingError: invalid connection option "pool"`. This error keeps forcing the cl-django container to exit, but the other containers are fine. I tried deleting the containers and setting them up again as directed by the developer instructions, but the error is persisting. Any thoughts?

**Comment by mlissner** [comment]
We upgraded the code to Django 5.1. It includes support for connection pools, so there are some new settings related to that. We deployed those yesterday, but they had issues in prod, so I backed out the changes. 

Try pulling the latest and I bet the problem will go away?

**Reply by v-anne** [reply]
Good now, thank you!
